js/timer.js

```diff
@@ -13,62 +13,62 @@ let timeInterval = null;
 let timerStatus = 'stopped';
 
 function stopWatch() {
-    seconds++;
-    if (seconds / 60 == 1) {
-        minutes++;
-        seconds = 0;
-        if (minutes / 60 == 1) {
-            hours++;
-            minutes = 0;
-        }
+  seconds++;
+  if (seconds / 60 == 1) {
+    minutes++;
+    seconds = 0;
+    if (minutes / 60 == 1) {
+      hours++;
+      minutes = 0;
     }
+  }
 
-    if (seconds < 10) {
-        leadingSeconds = '0' + seconds.toString();
-    } else {
-        leadingSeconds = seconds;
-    }
+  if (seconds < 10) {
+    leadingSeconds = '0' + seconds.toString();
+  } else {
+    leadingSeconds = seconds;
+  }
 
-    if (minutes < 10) {
-        leadingMinutes = '0' + minutes.toString();
-    } else {
-        leadingMinutes = minutes;
-    }
+  if (minutes < 10) {
+    leadingMinutes = '0' + minutes.toString();
+  } else {
+    leadingMinutes = minutes;
+  }
```

Comment:
Instead of using if and else loops, try using Ternary operator here this will reduce length of the code.